third_party/crashpad/third_party/googletest/BUILD.gn

```diff
@@ -0,0 +1,372 @@
+# Copyright 2017 The Crashpad Authors. All rights reserved.
```

Comment:
I guess there's not a good way to show this file as branched from `third_party/crashpad/third_party/gtest/BUILD.gn`?

Reply:
This is what we get for using `git-subtree.sh`.